orangewidget/tests/base.py

```diff
@@ -689,5 +691,25 @@ def open_widget_classes():
         yield
 
 
+@contextmanager
```

Comment:
This code is moved from orangewidget/tests/test_settings_handler.py to here, so it can be shared between test files.

The only change is the additional argument `handler`.